### PR TITLE
[Term Entry] Python:NumPy built-in-functions Matrix Determinant

### LINKED ISSUE #4481: [Term Entry] Python:NumPy built-in-functions Matrix Determinant
### Reason/inspiration (optional)

"We would like a new term entry on the `built-in-functions` term for [Numpy](https://www.codecademy.com/resources/docs/numpy): Matrix Determinant. The entry should go in a new file under `docs/content/numpy/concepts/built-in-functions/terms/determinant/determinant.md`.

The entry should describe the term and how it is used within statistical analysis. Also include: 
- An "Example" section that explains readers how to use the linalg.det function to find the determinat of a matrix.
- Should contain informations for functions and syntax.
- Should have a python codebyte with working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by boffskbob**
Created numpy determinant markdown file and folder, completed first revision of determinant.md

### Description

Closes #4481

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/numpy/concepts/built-in-functions/terms/determinant/determinant.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Syntax
+
+```pseudo
+numpy.linalg.det(array)
+```
+
+`.linalg.det()` provides the following argument:
+
+ - `array` : The numpy array_like object to compute the determinant for
```
Comment (letsbuildanything) [inline content/numpy/concepts/built-in-functions/terms/determinant/determinant.md:28]:
```suggestion
- `array` : The numpy array-like object for which the determinant is computed.
```

File: content/numpy/concepts/built-in-functions/terms/determinant/determinant.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Example
+
+The following example creates multiple numpy arrays then computes and returns the respective determinants.
```
Comment (letsbuildanything) [inline content/numpy/concepts/built-in-functions/terms/determinant/determinant.md:32]:
```suggestion
The following examples demonstrate the use of `.det()` method:
```

File: content/numpy/concepts/built-in-functions/terms/determinant/determinant.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+det_a = np.linalg.det(a)
+print(det_a)
+
+b = np.array([[5, 6], [7, 8]])
+det_b = np.linalg.det(b)
+print(det_b)
+
+c = np.array([[1, 2, 3], [0, 1, 4], [5, 6, 0]])
+det_c = np.linalg.det(c)
+print(det_c)
```
Comment (letsbuildanything) [inline content/numpy/concepts/built-in-functions/terms/determinant/determinant.md:47]:
```suggestion
mat_2x2 = np.array([[1, 2], [3, 4]])
det_mat_2x2 = np.linalg.det(mat_2x2)
print("Determinant of 2 x 2 matrix:", det_mat_2x2)  # prints output approx. -2.0

mat_3x3 = np.array([[1, 2, 3], [0, 1, 4], [5, 6, 0]])
det_mat_3x3 = np.linalg.det(mat_3x3)
print("Determinant of 3 x 3 matrix:", det_mat_3x3) # prints output approx. 1.0
```

File: content/numpy/concepts/built-in-functions/terms/determinant/determinant.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+This produces the following output:
+
+```shell
+-2.0
+-2.0
+1.0
```
Comment (letsbuildanything) [inline content/numpy/concepts/built-in-functions/terms/determinant/determinant.md:55]:
```suggestion
Determinant of 2 x 2 matrix: -2.0000000000000004
Determinant of 3 x 3 matrix: 0.9999999999999964
```

File: content/numpy/concepts/built-in-functions/terms/determinant/determinant.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+```
```
Comment (letsbuildanything) [inline content/numpy/concepts/built-in-functions/terms/determinant/determinant.md:56]: Please add one codebyte example here.

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@boffskbob I've suggested a few changes. Please make them.
Please remove any trailling spaces after punctuation. Otherwise, You did a great job!

**Comment by boffskbob** [comment]
@letsbuildanything revisions have been made!

**Review by letsbuildanything — APPROVED** [review]

**Review by Sriparno08 — APPROVED** [review]
Looks good 👍

**Pull request merged**